include/mfd/rp1_platform.h

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+/*
+ * Copyright (c) 2024 EPAM Systems
+ *
+ * Derived from linux rp1_platform.h
+ * Copyright (c) 2021-2022 Raspberry Pi Ltd.
+ */
+
+#ifndef _RP1_PLATFORM_H
+#define _RP1_PLATFORM_H
+
+#include <linux/types.h>
+#include <asm/byteorder.h>
+
+#define RP1_B0_CHIP_ID 0x10001927
```

Comment:
I saw the same in rp1.c

Reply:
removed rp1_platform.h. It was moved from linux source code, but I see no reason to leave it here